src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,100 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
+@SuppressWarnings({"rawtypes"})
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+    private int capacity;
+    private int size;
+    private Node[] nodes;
```

Comment:
```suggestion
    private Node<K, V>[] nodes;
```
use generic types here